integrations/postgres-integration.md

```diff
@@ -0,0 +1,131 @@
+# Kubecost / Postgres Integration
+
+{% hint style="info" %}
+This feature is only supported for Kubecost Enterprise.
+{% endhint %}
+
+Often times, users prefer to gather all their data and metrics in a single data warehouse. From there, they can run queries and generate reports to analyze the data across multiple sources. The Postgres Integration feature enables users to periodically query and export their Kubecost data to a central Postgres database.
+
+## Example use cases
+
+- Customized dashboarding used by FinOps/Analytics teams.
+- Single view populated by multiple data sources, to be used by developer/finance/operations teams.
+- Customized budgeting and forecasting via an in-house data pipeline.
+
+## Usage
+
+Prerequisites:
+
+- A running Postgres instance that is reachable by Kubecost.
+- Credentials for a Postgres user which has `CREATE`, `INSERT`, and `UPDATE` permissions on the database.
+
+### Step 1: Configure Helm values
+
+The below YAML is an example of how to configure the Postgres integration in your Helm values file. Notice that there are four different cronjobs being created (2 allocation queries, 1 asset query, 1 cloudCost query). Each cronjob is configured to run every `12h` and write to their respective tables in the Postgres database.
```

Comment:
While the internal implementation here is a cronjob, I'd just state the user-apparent aspect which just scheduled runs. Minor point though.

Reply:
I've updated the wording to say "query" instead of "cronjob". Does that work better?